ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/tasks/workload.yml

```diff
@@ -52,6 +52,42 @@
     - r_gitea_deployment.resources[0].status.availableReplicas is defined
     - r_gitea_deployment.resources[0].status.availableReplicas | int == r_gitea_deployment.resources[0].spec.replicas | int
 
+- name: search for gitea pod
```

Comment:
I'd put all that into a block under a

when: ocp4_workload_gitea_operator_install_admin_user | bool

condition.